docs/user_guide.rst

```diff
@@ -1,2 +1,28 @@
 User Guide
 ==========
+
+.. contents:: :depth: 3
+
+Example
+-------
+
+This is an example for setting up FoxPuppet and running a simple check to see if the browser window is private::
+
+    from foxpuppet import FoxPuppet
+    from selenium.webdriver import Firefox
+
+
+    selenium = Firefox(
+        firefox_binary='path/to/firefox-bin'
+    )
```

Comment:
Let's simplify this to `selenium = Firefox()` to just pick up the default location.